mw-webapp/src/component/image/Image.module.scss

```diff
@@ -1,3 +1,15 @@
+.imageWrapperFullScreen {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  padding: 7%;
```

Comment:
don't use %

Reply:
Ok

Reply:
Pease in comments write questions , suggestions, or status : done, deleted etc. Ok is not clear

Reply:
@Ekaterina1994 changed